nservicebus/tools/migrate-to-native-delivery.md

```diff
@@ -266,3 +308,23 @@ To list the status of timeouts for an a previous/in-progress run take the `Migra
 `SELECT * FROM TimeoutData_migration_{MigrationRunId}`
 
 This will show all the timeouts and to which batch they belong and also that status of that batch, `0=Pending`, `1=Staged` and `2=Completed`.
+
+### Azure Storage persistence
+
+To list the history and status of migrations open up the `timeoutsmigrationtoolstate` Azure Table inside the Storage Explorer.
+
+The migrated timeouts as well as their status can be inspected in the `timeoutsmigration` Azure Table. This will show all the timeouts and to which batch they belong and also that status of that batch, `0=Pending`, `1=Staged` and `2=Completed`.
+
+To list all timeouts that have been transferred from the endpoints timeout table into the `timeoutsmigration` table execute the following query and adjust the partition key cut off time accordingly. In order to avoid querying the additional timeout data inside the table scope the partition key to cutoff time plus a hundred years.
```

Comment:
```suggestion
To list all timeouts that have been transferred from the endpoints timeout table into the `timeoutsmigration` table, execute the following query and adjust the partition key cut-off time accordingly. In order to avoid querying the additional timeout data inside the table, scope the partition key to cutoff time plus a hundred years.
```